### PR TITLE
refactor(editor): Fix TypeScript issues in views (no-changelog)

### PR DESCRIPTION
**Pull request by elsmr**
## Review / Merge checklist
- [ ] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: packages/editor-ui/src/stores/ui.store.ts [hunk 1]
```diff
@@ -266,7 +266,7 @@ export const useUIStore = defineStore(STORES.UI, {
 						},
 					},
 				},
-			};
+			} as const;
```
Comment (mutdmour) [inline packages/editor-ui/src/stores/ui.store.ts:269]: better solution than what i tried 😍 Will close that PR now

File: packages/editor-ui/src/views/WorkflowHistory.vue [hunk 2]
```diff
@@ -116,11 +114,17 @@ onBeforeMount(async () => {
 	}
 });
 
+const normalizeSingleRouteParam = (name: string): string => {
```
Comment (mutdmour) [inline packages/editor-ui/src/views/WorkflowHistory.vue:117]: 😍

File: packages/editor-ui/src/views/WorkflowOnboardingView.vue [hunk 3]
```diff
@@ -26,10 +27,10 @@ const openWorkflowTemplate = async (templateId: string) => {
 			interpolate: { name: template.name },
 		});
 
-		const workflow = await workfowStore.createNewWorkflow({
+		const workflow = await workflowsStore.createNewWorkflow({
 			name,
 			connections: template.workflow.connections,
-			nodes: template.workflow.nodes,
+			nodes: template.workflow.nodes as INode[],
```
Comment (mutdmour) [inline packages/editor-ui/src/views/WorkflowOnboardingView.vue:33]: Is there a way to avoid typecasting here?

**Review by mutdmour — COMMENTED** [review]
Looks awesome. One question

File: packages/editor-ui/src/views/WorkflowOnboardingView.vue (hunk 3 above)
Comment (elsmr) [reply packages/editor-ui/src/views/WorkflowOnboardingView.vue]: done!

**Review by mutdmour — APPROVED** [review]
Thanks for addressing my feedback

**Pull request merged**